api/csi.go

```diff
@@ -81,18 +81,18 @@ const (
 type CSIVolume struct {
 	ID             string
 	Namespace      string
+	Name           string
+	ExternalID     string
 	Topologies     []*CSITopology
 	AccessMode     CSIVolumeAccessMode
 	AttachmentMode CSIVolumeAttachmentMode
 
 	// Combine structs.{Read,Write,Past}Allocs
 	Allocations []*AllocationListStub
 
-	// Healthy is true iff all the denormalized plugin health fields are true, and the
-	// volume has not been marked for garbage collection
-	Healthy             bool
-	VolumeGC            time.Time
```

Comment:
This was intended to mark volumes for automatic garbage collection, but we've moved that out of scope for now